src/routes/download/preview.svelte

```diff
@@ -6,6 +6,6 @@
 	<Metadata title="Files • Download Preview" image="download" />
 	<meta
 		http-equiv="refresh"
-		content="0; url=ms-appinstaller:?source=https://filescommunity.blob.core.windows.net/files/preview/Files.Package.appinstaller"
+		content="https://filescommunity.blob.core.windows.net/files/preview/Files.Package.appinstaller download"
```

Comment:
Why do you have `download` here and not in the other file?

Reply:
It doesn't seem to have made a difference.